interfaces/builtin/wayland.go

```diff
@@ -55,6 +55,9 @@ capability sys_tty_config,
 owner /run/user/[0-9]*/wayland-[0-9]* rwk,
 # Allow access to common client Wayland sockets from non-snap clients
 /run/user/[0-9]*/{mesa,mutter,sdl,wayland-cursor,weston,xwayland}-shared-* rw,
+# Some Wayland based toolkits (Qt, GTK3, SDL2) create shm files to pass cursor images
+# to the server. Although they are passed by FD we still need rw access to the file.
+/run/user/[0-9]*/snap.*/wayland-cursor-shared-* rw,
```

Comment:
should this say `owner ...`?

Reply:
> should this say `owner ...`?

I don't think that gains much (other rules don't specify "owner" either) and _could_ be a limitation. (When `su`ing to another user before starting a client application this would prevent it appearing on a confined desktop.)